runOpenVpn.sh

```diff
@@ -5,14 +5,37 @@ then
 	if [ -f /etc/openvpn/"${OPEN_VPN_CONFIG}".ovpn ]
   	then
 		echo "Starting OpenVPN using config ${OPEN_VPN_CONFIG}.ovpn"
-		exec openvpn --config /etc/openvpn/"${OPEN_VPN_CONFIG}".ovpn
+		OPEN_VPN_CONFIG=/etc/openvpn/${OPEN_VPN_CONFIG}.ovpn
 	else
 		echo "Supplied config ${OPEN_VPN_CONFIG}.ovpn could not be found."
 		echo "Using default OpenVPN gateway: Netherlands"
-		exec openvpn --config /etc/openvpn/Netherlands.ovpn
+		OPEN_VPN_CONFIG=/etc/openvpn/Netherlands.ovpn
 	fi
 else
 	echo "No VPN configuration provided. Using default: Netherlands"
-	exec openvpn --config /etc/openvpn/Netherlands.ovpn
+	OPEN_VPN_CONFIG=/etc/openvpn/Netherlands.ovpn
 fi
 
+# override resolv.conf
+if [ "$RESOLV_OVERRIDE" != "**None**" ];
```

Comment:
Ah ok, did not know that. Just thought it looked verbose. But, then it has a purpose :)